modules/platforms/dotnet/Apache.Ignite.Core/Configuration/DataStorageConfiguration.cs

```diff
@@ -143,16 +143,6 @@ public class DataStorageConfiguration
         /// </summary>
         public const bool DefaultWalCompactionEnabled = false;
 
-        /// <summary>
-        /// Default size of a memory chunk reserved for system cache initially.
-        /// </summary>
-        public const long DefaultSystemRegionInitialSize = 40 * 1024 * 1024;
```

Comment:
**Breaking change.**

Please mark `[Obsolete]` instead of removing a part of public API.